Makefile

```diff
@@ -1,5 +1,12 @@
 run-system-tests:
-	echo "TODO: write complete system tests"
+	@echo "Start OpenSlides Dev"
+	docker-compose -f docker/docker-compose.dev.yml up -d
+	@echo "Wait  until the system is actually up"
+	sleep 120
```

Comment:
@gsiv maybe you have an idea for that.
What I would need to run the system tests is to "ensure" here that `docker-compose -f docker/docker-compose.dev.yml up -d` was started in a way the the client and the proxy is actually "up and running".

In this case, up and running would imply that the system is actually reachable. This is not necessarily the case when `docker-compose -f docker/docker-compose.dev.yml up -d` finished the first time. Docker still needs some time untill the proxy (caddy?) Actully delivers the client.
Doing this immediately after docker compose finished would return "bad gateway" upon requesting asking `localhost:8000`